config.py

```diff
@@ -3,7 +3,12 @@
 
 settings = Dynaconf(
     envvar_prefix="FERRIS_ELF",
-    settings_files=['settings.toml', '.secrets.toml'],
+    settings_files=[
+        'settings.toml', 
+        '.secrets.toml',
+        '/srv/ferris-elf/config/settings.toml',
+        '/srv/ferris-elf/config/.secrets.toml'
```

Comment:
QA + Prod need these two extra lines to keep env config out of the way of git. This change will enable us to discard any spurious changes that do show up in the codebase somehow (ie, hacking) whenever we check for updates.